pyvo/io/uws/tree.py

```diff
@@ -274,9 +274,7 @@ def jobs(self, iterator, tag, data, config, pos):
 
 
 class Parameters(UWSElement, HomogeneousList):
-    """
-    Parameters element of a job
-    """
+    """Parameters element of a job"""
```

Comment:
My suspicion is that `Parameters` are recognized as a special heading and thus being complained about. Try to rephrase, maybe even saying `The parameters element of a job` would be enough.

Reply:
Okay yeah, I think so too.  I just tried escaping it, although maybe your way is better of just rephrasing it.

Reply:
Okay tried rephrasing it but it still seems to be trying to pick it up as a keyword.  I figure there has to be some way of escaping it but I'm not sure what it is...